Adafruit_MQTT.cpp

```diff
@@ -171,7 +171,7 @@ int8_t Adafruit_MQTT::connect() {
     if (subscriptions[i] == 0) continue;
 
     boolean success = false;
-    for (uint8_t retry=0; (retry<3) && !success; retry++) { // retry until we get a suback    
```

Comment:
Can't get my IDE to leave some of these newline ones alone - think someone committed using carriage returns so I'm just leaving them. Will open it up in another editor if it's an issue.

Reply:
Ok, shouldn't be a problem.